app-modules/etno/src/Models/Item.php

```
@@ -269,44 +269,48 @@ public static function documentRelations(array $with, ?\Closure $callback = null
+            AccessRights::published(),
+        );
+    }
 
-            return [$value->{$key}];
-        };
+    public function scopePublished(Builder $query): void
+    {
+        $query->where(function (Builder $query) {
+            $query->where(function (Builder $q) {
+                $q->whereJsonContains('etno_items.document_overrides', 'access_rights')
+                    ->whereIn('etno_items.access_rights', AccessRights::published());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Etno\Enums\AccessRights' in method 'scopePublished'. (undefined)

(StaticAccess)